integration_tests/src/main/python/delta_zorder_test.py

```diff
@@ -113,11 +114,13 @@ def optimize_table(spark):
 # This test is very similar to `test_dpp_reuse_broadcast_exchange` but it tests joining using a Z-ordered
 # column
 @delta_lake
+@allow_non_gpu('CollectLimitExec')
 @ignore_order(local=True)
 @pytest.mark.skipif(not is_databricks104_or_later(), reason="Dynamic File Pruning is only supported in Databricks 10.4+")
 @pytest.mark.parametrize('s_index', list(range(len(_statements))), ids=idfn)
 @pytest.mark.parametrize('aqe_enabled', ['false', 'true'])
-def test_delta_dfp_reuse_broadcast_exchange(spark_tmp_table_factory, s_index, aqe_enabled):
+@pytest.mark.parametrize("deletion_vector_conf", deletion_vector_conf, ids=idfn)
```

Comment:
Do we want to raise an issue for this failure (`ShuffleExchangeExec` falling back to GPU), and link this here with `deletion_vector_values_with_reasons`?